.gitignore

```diff
@@ -96,3 +96,4 @@ src/TestResults
 test/core/customast/test.txt
 test/core/files/test.png
 test/core/migration/writetext.txt
+src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs
```

Comment:
I'm not sure, but I believe there is a reason we don't have this as part of the gitignore, you can simply not commit changes to this file until we dig deeper.